package.json

```diff
@@ -50,6 +50,7 @@
     "eslint-config-stripe": "^1.0.10",
     "eslint-plugin-flowtype": "^2.32.1",
     "eslint-plugin-jest": "^19.0.1",
+    "eslint-plugin-jsx-a11y": "2.2.3",
```

Comment:
i couldn't get my local eslint working with atom with the 2.x.x version (related to -> (https://github.com/evcohen/eslint-plugin-jsx-a11y/issues/116)  . May have been my local setup. feel free to remove it.